SUPPORTED_APIS.md

```diff
@@ -2,31 +2,63 @@
 
 > ⚠️ **WORK IN PROGRESS**: This list is non-comprehensive and being updated
 
-# Key
+## Supported APIs Table
+
+| Category | API | Status | Description |
```

Comment:
Don't know if `Category` is needed here, but if we want to keep it, can we please rename to `Namespace` and turn all values in column from `CONFIG NAMESPACE` -> `CONFIG`?

Reply:
Great suggestion